hack/release.sh

```diff
@@ -12,5 +12,5 @@ ${TOOLBIN}/yq eval --inplace ".global.tag = \"$RELEASE\"" ./charts/data-movement
 ${TOOLBIN}/yq eval --inplace ".version = \"$RELEASE\"" ./modules/fybrik-implicit-copy-batch/Chart.yaml
 ${TOOLBIN}/yq eval --inplace ".version = \"$RELEASE\"" ./modules/fybrik-implicit-copy-stream/Chart.yaml
 
-${TOOLBIN}/yq eval --inplace ".image = \"ghcr.io/fybrik/mover:$RELEASE\"" modules/fybrik-implicit-copy-batch/values.yaml
-${TOOLBIN}/yq eval --inplace ".image = \"ghcr.io/fybrik/mover:$RELEASE\"" modules/fybrik-implicit-copy-stream/values.yaml
+${TOOLBIN}/yq eval --inplace ".spec.chart.name = \"ghcr.io/fybrik/fybrik-implicit-copy-batch:$RELEASE\"" ./modules/implicit-copy-batch-module.yaml
```

Comment:
did we talk to remove these 2 lines?

Reply:
I removed them and added two lines to update the `implicit-copy-batch-module.yaml` and `implicit-copy-stream-module.yaml` files.